core/src/main/java/nu/yona/server/subscriptions/entities/BuddyConnectResponseMessage.java

```diff
@@ -58,24 +46,22 @@ public void setProcessed()
 	}
 
 	public static BuddyConnectResponseMessage createInstance(UUID respondingUserID, UUID respondingUserAnonymizedID,
-			UUID destinationID, String nickname, String message, UUID buddyID, BuddyAnonymized.Status status)
+			String nickname, String message, UUID buddyID, BuddyAnonymized.Status status)
 	{
 		return new BuddyConnectResponseMessage(UUID.randomUUID(), respondingUserID, respondingUserAnonymizedID, nickname, message,
-				buddyID, destinationID, status);
+				buddyID, status);
 	}
 
 	@Override
 	public void encrypt(Encryptor encryptor)
```

Comment:
This method can now be removed, as there is nothing encrypted in this class anymore.